Frontend/ui-customer/src/Screens/CustomerUtils/ReportIssueChatScreen/View.purs

```diff
@@ -0,0 +1,484 @@
+-}
+
+module Screens.ReportIssueChatScreen.View where
+
+import Data.Maybe
+
+import Animation (screenAnimation, screenAnimationFadeInOut)
+import Common.Types.App (LazyCheck(..))
+import Components.AddAudioModel (view) as AddAudioModel
+import Components.AddImagesModel (view) as AddImagesModel
+import Components.ChatView as ChatView
+import Components.PrimaryButton.View (view) as PrimaryButton
+import Components.PrimaryEditText as PrimaryEditText
+import Components.RecordAudioModel.View (view) as RecordAudioModel
+import Components.ViewImageModel.View (view) as ViewImageModel
+import Data.Array (length, null)
+import Data.String (length, trim) as STR
+import Debug (spy)
+import Effect (Effect)
+import Effect.Uncurried (runEffectFn2)
+import Engineering.Helpers.Commons as EHC
+import Font.Size (a_16, a_20, a_14, a_17, a_18) as FontSize
+import Font.Style  as FontStyle
+import Helpers.Utils (fetchImage, FetchImageFrom(..))
+import JBridge (storeCallBackImageUpload, storeCallBackUploadMultiPartData, storeKeyBoardCallback) as JB
+import Language.Strings (getString)
+import Language.Types (STR(..))
+import Prelude (Unit, bind, const, pure, show, unit, ($), (-), (<<<), (<>), (>), (||), (==), not)
+import PrestoDOM (background, frameLayout, imageView, linearLayout, relativeLayout, textView)
+import PrestoDOM.Events (afterRender, onBackPressed, onClick)
+import PrestoDOM.Properties (adjustViewWithKeyboard, alignParentBottom, alpha, background, color, cornerRadii, cornerRadius, fontStyle, gravity, height, imageUrl, imageWithFallback, layoutGravity, lineHeight, margin, maxWidth, orientation, padding, position, stroke, text, textSize, visibility, weight, width)
+import PrestoDOM.Types.Core (Corners(..), Gravity(..), Length(..), Margin(..), Orientation(..), Padding(..), Position(..), PrestoDOM, Screen, Visibility(..))
+import Screens.ReportIssueChatScreen.ComponentConfig (cancelButtonConfig, doneButtonConfig, primaryEditTextConfig, viewImageModelConfig, addImageModelConfig, recordAudioModelConfig, addAudioModelConfig, chatConfig)
+import Screens.ReportIssueChatScreen.Controller (Action(..), ScreenOutput, eval)
+import Screens.Types (ReportIssueChatScreenState)
+import Styles.Colors (black700, black800, black900, black9000, blue900, brightBlue, blueTextColor, grey700, grey900, greyLight, lightGreyBlue, white900) as Color
+
+screen :: ReportIssueChatScreenState -> Screen Action ReportIssueChatScreenState ScreenOutput
+screen initialState =
+  { initialState
+  , view
+  , name: "ReportIssueChatScreen"
+  , globalEvents:
+      [ ( \push -> do
+            _ <- push ShowOptions 
+            if (not initialState.props.initalizedCallbacks) then do
+                _ <- JB.storeCallBackImageUpload push ImageUploadCallback
+                _ <- runEffectFn2 JB.storeCallBackUploadMultiPartData push UploadMultiPartDataCallback
+                _ <- runEffectFn2 JB.storeKeyBoardCallback push KeyboardCallback
```

Comment:
create a function which executes all the function inside global events and call that function alone inside global events. Refer to TicketBookingScreen in Customer for this.